api/v1/storagecluster_types.go

```diff
@@ -305,6 +341,9 @@ type StorageClusterStatus struct {
 	// ExternalSecretHash holds the checksum value of external secret data.
 	ExternalSecretHash string `json:"externalSecretHash,omitempty"`
 
+	// ExternalStorage shows the status of the external cluster
+	ExternalStorage ExternalStorageClusterStatus `json:"externalStorage,omitempty"`
```

Comment:
I don't think this is something we would like to support? What value does it bring? what is the use case?